app/src/main/res/layout/row_issue_template.xml

```diff
@@ -3,7 +3,7 @@
     android:orientation="vertical"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:paddingHorizontal="@dimen/content_padding"
+    android:paddingHorizontal="24dp"
```

Comment:
Because the list elements are not aligned with the dialog title, as you can see below.
I assume that the extra padding is needed because we are using custom views for those elements.

![photo1644599852](https://user-images.githubusercontent.com/30041551/153708658-860c2e5a-cad8-4e05-9926-f63dd2f955cf.jpeg)